### PR TITLE
Refresh the quick starts sample to use updated sdk.

### PR DESCRIPTION
**Pull request by moirf**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: ServerRecording/src/main/java/com/acsrecording/api/Controller/CallRecordingController.java [hunk 1]
```diff
@@ -146,10 +143,10 @@ public StartCallRecordingResult startRecordingWithArgs(String serverCallId, Stri
             recordingDataMap.replace(serverCallId, output.getRecordingId());
 
             return output;
-        } catch (URISyntaxException e) {
+        } catch (Exception e) {
             e.printStackTrace();
-            return null;
-        }
+            return  null;
```
Comment (chrwhit) [inline ServerRecording/src/main/java/com/acsrecording/api/Controller/CallRecordingController.java:148]: nit: extra space

**Review by chrwhit — APPROVED** [review]

File: ServerRecording/pom.xml [hunk 2]
```diff
@@ -45,32 +45,32 @@
 		<dependency>
 			<groupId>com.azure</groupId>
 			<artifactId>azure-identity</artifactId>
-			<version>1.3.5</version>
+			<version>1.5.4</version>
 		</dependency>
 		<dependency>
 			<groupId>com.azure</groupId>
 			<artifactId>azure-communication-callingserver</artifactId>
```
Comment (williamzhao87) [inline ServerRecording/pom.xml:52]: Can you please update package reference to azure-communication-callautomation?  Package has been renamed.

**Review by williamzhao87 — CHANGES_REQUESTED** [review]

File: CallPlayAudio/pom.xml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      <exclusions>
+        <exclusion>
+          <groupId>com.azure</groupId>
+          <artifactId>azure-communication-common</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-communication-callingserver</artifactId>
```
Comment (sarkar-rajarshi) [inline CallPlayAudio/pom.xml:70]: Can you please update package reference to azure-communication-callautomation? Package has been renamed.

File: OutboundCallReminder/pom.xml [hunk 4]
```diff
@@ -70,22 +70,22 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-callingserver</artifactId>
```
Comment (sarkar-rajarshi) [inline OutboundCallReminder/pom.xml:72]: same here

File: CallPlayAudio/pom.xml (hunk 3 above)
Comment (moirf) [reply CallPlayAudio/pom.xml:70]: ok, will update.

**Review by Paresh-Arvind-Patil — APPROVED** [review]

File: CallPlayAudio/readme.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+This sample application shows how the Azure Communication Services Server Calling SDK can be used to build IVR related solutions. This sample makes a call to a phone number or a communication identifier and plays an audio message when call is connected.
+## Getting started
+
+### Prerequisites
+
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/)
+- [Java Development Kit (JDK) version 11 or above](https://docs.microsoft.com/azure/developer/java/fundamentals/java-jdk-install)
+- [Apache Maven](https://maven.apache.org/download.cgi)
+- Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You'll need to record your resource **connection string** for this sample.
+- Get a phone number for your new Azure Communication Services resource. For details, see [Get a phone number](https://docs.microsoft.com/azure/communication-services/quickstarts/telephony-sms/get-phone-number?pivots=platform-azp)
```
Comment (cochi2) [inline CallPlayAudio/readme.md:21]: This link gives me 404

File: CallPlayAudio/src/main/java/com/communication/CallPlayAudio/Ngrok/NgrokService.java [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    }
+
+    /// <summary>
+    /// Ensures that NGROK is not running.
+    /// </summary>
+    private void ensureNgrokNotRunning() {
+        BufferedReader input;
+        String ngrokProcess = "ngrok.exe";
+        String processFilter = "/nh /fi \"Imagename eq " + ngrokProcess + "\"";
+        String tasksCmd = System.getenv("windir") + "/system32/tasklist.exe " + processFilter;
```
Comment (cochi2) [inline CallPlayAudio/src/main/java/com/communication/CallPlayAudio/Ngrok/NgrokService.java:29]: Windows is not an explicit requirement for your sample (according to the readme file).
Can we make this extensible to other operating systems?

File: CallPlayAudio/pom.xml [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-communication-callautomation</artifactId>
+      <version>1.0.0-alpha.20221017.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.microsoft.cognitiveservices.speech</groupId>
+      <artifactId>client-sdk</artifactId>
+      <version>1.12.1</version>
```
Comment (cochi2) [inline CallPlayAudio/pom.xml:76]: also update to 1.24.0

File: CallPlayAudio/src/main/java/com/communication/CallPlayAudio/CallPlayTerminate.java [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+public class CallPlayTerminate {
+
+    private final CallConfiguration callConfiguration;
+    private final CallAutomationClient callingAutomationClient;
+    private CallConnection callConnection = null;
```
Comment (cochi2) [inline CallPlayAudio/src/main/java/com/communication/CallPlayAudio/CallPlayTerminate.java:33]: this is never set and throws NPE

File: CallPlayAudio/src/main/java/com/communication/CallPlayAudio/App.java [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+        try {
+            if (key != null && !key.isEmpty() && region != null && !region.isEmpty() && customMessage != null
+                    && !customMessage.isEmpty()) {
+                SpeechConfig config = SpeechConfig.fromSubscription(key, region);
+                config.setSpeechSynthesisOutputFormat(SpeechSynthesisOutputFormat.Riff16Khz16BitMonoPcm);
+
+                AudioConfig audioConfig = AudioConfig.fromWavFileInput(
+                        "src/main/java/com/communication/CallPlayAudio/audio/custom-message.wav");
+                SpeechSynthesizer synthesizer = new SpeechSynthesizer(config, audioConfig);
+                synthesizer.SpeakTextAsync(customMessage);
```
Comment (cochi2) [inline CallPlayAudio/src/main/java/com/communication/CallPlayAudio/App.java:142]: it's understandable that you want to execute stuff while Speech is being generated, but this is not working. 
My theory is that the `synthesizer` is being killed with `synthesizer.close()` before being actually executed.
I'd recommend `SpeakText(customMessage)` or `SpeakTextAsync(customMessage).get()`

File: CallPlayAudio/src/main/java/com/communication/CallPlayAudio/Ngrok/NgrokService.java (hunk 6 above)
Comment (moirf) [reply CallPlayAudio/src/main/java/com/communication/CallPlayAudio/Ngrok/NgrokService.java:29]: will look in later versions

File: OutboundCallReminder/pom.xml [hunk 10]
```diff
@@ -69,23 +69,23 @@
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-communication-callingserver</artifactId>
-      <version>1.0.0-beta.3</version>
+      <artifactId>azure-communication-callautomation</artifactId>
+      <version>1.0.0-alpha.20221017.1</version>
```
Comment (sarkar-rajarshi) [inline OutboundCallReminder/pom.xml:73]: Please use the beta version: `1.0.0-beta.1`

File: ServerRecording/pom.xml [hunk 11]
```diff
@@ -45,32 +45,32 @@
...
 			<artifactId>azure-identity</artifactId>
-			<version>1.3.5</version>
+			<version>1.5.4</version>
 		</dependency>
 		<dependency>
 			<groupId>com.azure</groupId>
-			<artifactId>azure-communication-callingserver</artifactId>
-			<version>1.0.0-beta.4</version>
+			<artifactId>azure-communication-callautomation</artifactId>
+			<version>1.0.0-alpha.20221017.1</version>
```
Comment (sarkar-rajarshi) [inline ServerRecording/pom.xml:53]: Please use the beta version: `1.0.0-beta.1`

**Pull request closed without merging**